src/ResourceManager/Dns/SetEnv.ps1

```diff
@@ -6,16 +6,16 @@
     [parameter(Mandatory=$false)]
     [string] $ServicePrincipalSecret,
     [parameter(Mandatory=$false)]
+    [string] $TenantId = "24c154bb-0619-4338-b30a-6aad6370ee14",
+    [parameter(Mandatory=$false)]
     [string] $Environment = "Prod" # Prod for arm prod , DOGFOOD for DOGFOOD
 )
 
-$NonProdTenant = "24c154bb-0619-4338-b30a-6aad6370ee14";
-
 $env:AZURE_TEST_MODE="Record";
 $str = "HttpRecorderMode=Record;SubscriptionId="+$SubscriptionId+";Environment="+$Environment+";ServicePrincipal="+$ServicePrincipal+";Password="+$ServicePrincipalSecret+";ServicePrincipalSecret="+$ServicePrincipalSecret ;
 if ($Environment -ine "Prod")
 {
-    $str+= ";AADTenant="+$NonProdTenant;
+    $str+= ";AADTenant="+$TenantId;
```

Comment:
Do not add bindary files to local feed - you will need to publish these packages before moving this into the September branch